contrib/confluence/src/python/pants/contrib/confluence/tasks/confluence_publish.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+import textwrap
+
+from pants.base.exceptions import TaskError
+from pants.task.task import Task
+from pants.util import desktop
+from pants.util.dirutil import safe_open
+
+from pants.contrib.confluence.targets.doc_page import Page
+from pants.contrib.confluence.util.confluence_util import Confluence, ConfluenceError
+
+
+# Copied from `pants.backend.docgen.tasks.confluence_publish`
+
+# TODO: Rethink this. We shouldn't require subclassing this. Instead, the wiki identity should come
```

Comment:
Is this portion now resolved? If not, it might be time for a ticket.

Reply:
I just copy-pasted everything into this directory. I was going to address this when I make the API changes. See https://github.com/pantsbuild/pants/issues/4993